.pipelines/pipeline.yaml

```diff
@@ -158,32 +158,49 @@ stages:
           name: "$(BUILD_POOL_NAME_LINUX_AMD64)"
         strategy:
```

Comment:
should we change the name of this job now that we also do windows builds in it?

Reply:
fixed